### ISSUE TITLE
The orange text bubble messages may be hard to read

### ISSUE THREAD

**Issue by DNin01** [LLK/scratch-www#7343]
### The Problem

With white text on a background with the color `#ffab1a`, the text contrast does not pass the WCAG guidelines.

### Steps to Reproduce

1. Sign in
2. Visit a project page
3. Click Report, choose a reporting reason, then click within the popup outside the text field or type a very short or very long message. A message will pop up asking you to fill out the field.

### How to Fix

Try black text.

### System Details

N/A

**Screenshots**
![An empty report form](https://user-images.githubusercontent.com/106490990/213306647-8f8042f1-acfb-483f-8b95-80ba2c06a72a.png)

**Comment by Secret-chest** [comment]
What about making it bold?

**Comment by DNin01** [comment]
> What about making it bold?

The WCAG guidelines would still apply.

**Comment by Secret-chest** [comment]
You know those numbers aren't exact?

**Comment by Secret-chest** [comment]
Please don't sacrifice usability for accesibility.

**Comment by Accio1** [comment]
> Please don't sacrifice usability for accesibility.

@Secret-chest What about usability for people with visual impairments who may have difficulty seeing text that does not have sufficient contrast against the background? Accessibility is important because it ensures that the site is able to be used by the widest range of people possible.

**Comment by Secret-chest** [comment]
@Accio1 But these changes must then be taken with care and appeal to as many people as possible, or be toggleable.
